plausible/umbrel-app.yml

```diff
@@ -0,0 +1,27 @@
+manifestVersion: 1
+id: plausible
+icon: https://svgshare.com/i/y30.svg
```

Comment:
This can actually be removed entirely. We will host the icon image here: https://github.com/getumbrel/umbrel-apps-gallery
```suggestion
```